app/widget.js

```diff
@@ -4,7 +4,8 @@ import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { createStore, applyMiddleware, compose } from 'redux';
 import thunk from 'redux-thunk';
-import { bootstrapWidgetData } from './actions';
+import actionTrigger, { ajaxWidgetData, listenerWidgetData } from './actions';
```

Comment:
it seemed a little odd to have the mixed exports from actions, with `actionTrigger` as default, then the rest.